src/orquestra/quantum/openfermion/linalg/sparse_tools.py

```diff
@@ -144,7 +147,73 @@ def jordan_wigner_sparse(fermion_operator, n_qubits=None):
     return sparse_operator
 
 
-def qubit_operator_sparse(qubit_operator, n_qubits=None):
+def pauli_sum_sparse(qubit_operator: PauliSum, n_qubits=None):
```

Comment:
If we need this (I'm not sure if we do 🤔 ) it should live in the `operators` dir, not in `openfermion`.

Reply:
This is used for getting the expectation value in `backend`. We will move it to the right place in a later PR when we move the operators out of `wip` and get rid of openfermion.